schema/2.0.0/seurat_encoding.md

```diff
@@ -220,7 +227,13 @@ Each available embedding is stored as a named element of the slot `reductions`.
     </tr>
     <tr>
       <th>Value</th>
-        <td><code>DimenReduc</code> object</td>
+        <td><code> SeuratObject::DimenReduc</code> object</td>
     </tr>
 </tbody></table>
 <br>
+
+The matrix with reductions is stored in the slot `cell.embeddings` of the `SeuratObject::DimenReduc`. Rows correspond to cells and are named with cell IDs; columns correspond to reductions and are named `[key]_[number]`, where `key` is an alphanumeric string and `number` is an integer, e.g. `PC_1`.
```

Comment:
is stored in the `cell.embeddings` slot of `SeuratObject::DimenReduc`. Rows correspond to cells and are named with cell IDs. Columns correspond to reductions and are named `[key]_[number]`, where `key` is an alphanumeric string and `number` is an integer, e.g. `PC_1`.